.github/workflows/tests.yml

```diff
@@ -6,6 +6,8 @@ name: Build and test
 on:
   pull_request:
   push:
+    branches:
+      - main
```

Comment:
Please revert as this will be very annoying to people sending pull requests: it would either force them to work on `main` or to not see their CI running before creating a pull request and you hitting the approve button.

Reply:
Are you sure about this? Without this, I get duplicate runs in this PR
![image](https://github.com/user-attachments/assets/da376d33-ed01-4e98-8eea-a7260e61d561)

Reply:
@mpartel yes, duplicate runs is the expected behavior; it's in fact two different commits: the head of the branch and a temporary merge commit into the target branch.

Reply:
Oh, ok, removed :+1:

Reply:
@mpartel thank you! :pray: